Makefile

```diff
@@ -168,30 +166,24 @@ install-lite:
 	# install nxcomp library
 	$(MAKE) -C nxcomp install
 
-	# install nxproxy wrapper script
-	$(INSTALL_DIR) $(DESTDIR)$(BINDIR)
-	sed -e 's|@@NXLIBDIR@@|$(NXLIBDIR)|g' bin/nxproxy.in > bin/nxproxy
-	$(INSTALL_PROGRAM) bin/nxproxy $(DESTDIR)$(BINDIR)
```

Comment:
Do we need a compat symlink here as well?

Reply:
> Nope, in fact the nxagent symlink should go ASAP, too.

Okay, `nxproxy` is historically difficult anyway, since sometimes a wrapper script was used and other times it wasn't (leading to the problems in the first place, really.)